assembl/views/api2/discussion.py

```diff
@@ -1613,6 +1617,13 @@ def phase1_csv_export(request):
         SENTIMENT_CREATION_DATE.encode('utf-8'),
     ]
 
+    extra_columns_info = load_social_columns_info(discussion, language)
+    if extra_columns_info:
+        # insert after email
```

Comment:
This is actually after sentiments count, not creator email.
Are we allowed to change the order of the fields here @ayyazdaniaryan ?
This phase1 export is used by bigdatext.

Reply:
corrected, thank you.